### DISCUSSION TITLE
how to check WebRTC is p2p or relay?

### DISCUSSION THREAD

**Discussion by Prajwal-infraon** [Ylianst/MeshCentral#5579] [Q&A]
Is there any way to know if the remote desktop connection is
established by WebRTC p2p or relay?

**Comment by NiceGuyIT** [comment]
I don't know if the software provides this information. Maybe check the open connections to see if there's one connected to the public IP of the other side? i.e. `ss` on Linux.

**Comment by si458** [comment]
if i remember correctly, the meshecntralrouter software doesnt support webrtc
but if the web panel is using webrtc, it will show 'webrtc' next to 'Connected'
![2022-05-15-13-58-29](https://github.com/Ylianst/MeshCentral/assets/765314/ddc1adce-32ea-4416-8c7c-50ecb820b452)
